R/shiny_fuse.R

```diff
@@ -330,6 +324,32 @@ shiny_fuse <- function(out_annofuse = NULL) {
         )
       }
     })
+    
+    output$export_controls <- renderUI({
+      if (is.null(values$annofuse_tbl)) {
+        return(NULL)
+      } else {
+        tagList(
+          menuItem(
+            "Plot export settings", 
+            icon = icon("images"),
+            startExpanded = TRUE,
+            numericInput(
+              inputId = "export_width",
+              label = "Export width (in)",
```

Comment:
Hahah - accommodating the US folks using inches 😉 .

Reply:
:D no clue how widely used are cm on that side of the big pond 😛